src/log-pcap.c

```diff
@@ -25,6 +25,10 @@
  * Pcap packet logging module.
  */
 
+#include <sys/types.h>
+#include <dirent.h>
```

Comment:
Oops, sys/types.h is not needed.  

Probably needs this for fnmatch as well (readdir is more likely to exist I believe).  What do we do if we don't have readdir? Just log a warning ring buffer won't pick up on restart?